src/renderer/components/MainPage.tsx

```diff
@@ -294,9 +294,9 @@ class MainPage extends React.PureComponent<Props, State> {
     }
 
     handleCloseDropdowns = () => {
+        console.log('MainPage.handleCloseDropdowns');
```

Comment:
```suggestion
        log.debug('MainPage.handleCloseDropdowns');
```

Dunno how this got past the linter :P